Index: Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Dockerfile b/Dockerfile
new file mode 100644
--- /dev/null	(date 1712671712926)
+++ b/Dockerfile	(date 1712671712926)
@@ -0,0 +1,14 @@
+# Use an official Java runtime as a parent image
+FROM openjdk:19
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY . /app
+
+# Build the application
+RUN javac MyApp.java
+
+# Run the application
+CMD ["java", "MyApp"]
